src/eva/tests/config/testMonDataSpaceHirs4Metop-A.yaml

```diff
@@ -26,7 +26,9 @@ graphics:
         figure size: [12,10]
         title: 'hirs4_metop-a | Channel 1 | Obs Count'
         output name: lineplots/hirs4_metop-a/hirs4_metop-a.0.count.png
-
+        plot logo:
```

Comment:
add logo? just a thought

Reply:
That would require change to emcpy. Is that what you want?

Reply:
in that case then redacted, it's just semantics after all

Reply:
Well the yaml could use "add logo" and then `figure_driver.py` could translate "add logo" to "plot_logo()".  No biggie.